helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestWagedRebalancer.java

```diff
@@ -423,15 +426,16 @@ public void testRebalanceOnChanges() throws IOException, HelixRebalanceException
     // assignment since there is only current state change.
     newIdealStates =
         rebalancer.computeNewIdealStates(clusterData, resourceMap, new CurrentStateOutput());
-    algorithmResult = _algorithm.getRebalanceResult();
+    Map<String, ResourceAssignment> newAlgorithmResult = _algorithm.getRebalanceResult();
 
     // Verify that only the changed resource has been included in the calculation.
-    validateRebalanceResult(Collections.emptyMap(), newIdealStates, algorithmResult);
-    // Both assignment state should be empty.
+    validateRebalanceResult(Collections.emptyMap(), newIdealStates, newAlgorithmResult);
+    // There is nothing changes the baseline since only current state changed.
```

Comment:
"There should be no changes in the baseline since only the currentStates changed"?